src/sparsezoo/analyze_v2/model_validators/distribution_analysis_model.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List, Optional, Union
+
+from pydantic import BaseModel, validator
+
+from sparsezoo.analyze_v2.model_validators.utils import type_validator
+
+
+class DistributionAnalysisModel(BaseModel):
+    counts: Optional[int]
+    mean: Optional[float]
+    median: Optional[float]
```

Comment:
because some nodes may not have weights, so instead of outputting zero, which means there is a value, i made it None, so when we compute the aggregated values it doesn't consider None as a value